js/renovation/ui/editors/check_box.tsx

```diff
@@ -16,11 +16,14 @@ import devices from '../../../core/devices';
 import Guid from '../../../core/guid';
 import { Widget } from '../common/widget';
 import BaseComponent from '../../component_wrapper/editors/check_box';
+import { normalizeStyleProp } from '../../../core/utils/style';
 import { BaseWidgetProps } from '../common/base_props';
 import { combineClasses } from '../../utils/combine_classes';
 import { EffectReturn } from '../../utils/effect_return.d';
 import { ValidationMessage } from '../overlays/validation_message';
 
+const ICON_FONT_SIZE_RATIO = 16 / 22;
```

Comment:
Is it ok for material theme where default value is 18 instead of 22?

Reply:
nope, working on fix